tooling/build/scripts/publishing/index.ts

```diff
@@ -270,7 +270,7 @@ function generateSitemapTree(
         title,
         permalink: `${pathPrefix.length === 1 ? "" : pathPrefix}/${danglingDirectory}`,
         lastModified: new Date().toISOString(),
-        layout: "index",
+        layout: folder?.type === "Collection" ? "collection" : "index",
```

Comment:
how come this was previously always index 🤔 wouldn't that have caused wrong layouts on collections?

Reply:
yea, we were always doing it manually so it didn't matter AHHA

Reply:
this is a bugfix (similar to the adding of index on prisma schema)